static/css/diff.less

```diff
@@ -115,4 +115,34 @@ ins {
 del {
   color: #e44028;
   text-decoration: line-through;
+}
+#back2Top {
+    width: 40px;
+    line-height: 40px;
+    overflow: hidden;
+    z-index: 999;
+    display: none;
+    cursor: pointer;
+    -moz-transform: rotate(270deg);
+    -webkit-transform: rotate(270deg);
+    -o-transform: rotate(270deg);
+    -ms-transform: rotate(270deg);
+    transform: rotate(270deg);
+    position: fixed;
+    bottom: 50px;
+    right: 0;
+    background-color: #DDD;
```

Comment:
you'll need to use a variable for this color. Running npm test should fail for this

Reply:
@lighter-grey appears to be the color you need here

Reply:
yes I found @lighter-grey in color.less and used it here. Thanks